installer.sh

```diff
@@ -77,7 +77,7 @@ case "$ID" in
                 PACKAGE_MGR=$(command -v yum)
```

Comment:
If RHEL 7 / CentOS 7 is still supported can you/we also change the above to `7*)` following my  comments from to issue https://github.com/keylime/keylime/issues/578

Reply:
Good idea, done

Reply:
The same is probably true for RHEL/ CentOS 8.